Makefile

```diff
@@ -26,12 +26,16 @@ ifeq ($(UNAME_S),Darwin)
 	endif
 endif
 
+ifneq ($(CC), gcc)
+	CC=gcc
```

Comment:
If the person running `make` really wants to use a different CC, wouldn’t this override them? Have you tried using CMake instead?